pkg/install/convertconfig.go

```diff
@@ -36,7 +36,7 @@ import (
 // as ClusterDeployment is used as a CRD API.
 //
 // It is assumed the caller will lookup the admin password and ssh key from their respective secrets.
```

Comment:
The "admin password" reference is stale now.  I'm not sure what to replace it with though, because I don't understand the whole "caller will lookup" idea.  The caller is passing these into `GenerateInstallConfig`, so why would they need to look them up at all?  And how do secrets come into this?

Reply:
In hive's ClusterDeployment the sshKey and pullSecret (and previously admin password) are referenced via secrets. Whatever code calls this needs to look them up, I didn't want this function responsible for talking to etcd.